.github/workflows/fuzzers.yml

```diff
@@ -33,6 +32,11 @@ jobs:
           - libfuzzer_nocmp
           - manul
           - mopt
+          # Greybox fuzzers
```

Comment:
Funny, I see regular AFL referred to as greybox (e.g. https://www.fuzzingbook.org/html/GreyboxFuzzer.html). I don't exactly know where that leaves QEMU mode (I call it blackbox, though this feels wrong too since it's more like regular AFL than fuzzers that completely ignore coverage).

Reply:
the common definition is : white = source code, grey = binary-only but system access (so it is possible to attach a debugger, reverse the binary etc), black = no access besides to the API (e.g. a TCP service on an embedded device that you only have TCP/IP access to). By that definition AFL is white (afl-gcc/afl-clang-fast) and grey (qemu, unicorn). :)

Reply:
TBF, the distinction between white- and grey-box is diminishing day by day. Initially, it was to distinguish between what analyses one can do on a target, thus white means one can do whatever analysis suits (full access to the internals on the target, which was possible if the source code was available) and grey means limited access (only observing execution traces as is possible when only binary code in available). But as there are tools to perform sophisticated analyses on binary code directly, we cease to have that distinction any more. For example: "SAGE: Whitebox Fuzzing for Security Testing" works on binaries directly!